{{cookiecutter.project_slug}}/Dockerfile

```
@@ -0,0 +1,21 @@
+# BASE
+FROM python:3.7.7 AS base
+
+# install poetry
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+# source poetry
+# https://stackoverflow.com/questions/59895745/poetry-fails-to-install-in-docker
+ENV PATH = "${PATH}:/root/.poetry/bin"
+
+WORKDIR /home/root/
+
+# dev with toml
+FROM base as dev
+COPY pyproject.toml .
+RUN poetry install &&\
+    rm pyproject.toml
```

```suggestion
COPY pyproject.toml poetry.lock* ./
RUN poetry install &&\
    rm pyproject.toml poetry.lock
```
this should make use of `poetry.lock`, if it exists - but should not fail if it does not exist.